<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskList</class>
 <widget class="QDialog" name="TaskList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QListView" name="completedListView">
     <property name="styleSheet">
      <string notr="true">QListView { font-size: 20pt; font-weight: bold; }
     QListView::item { background-color: #2ECC71; padding: 10%;
     border: 1px solid #27AE60; }
     QListView::item::hover { background-color: #27AE60 }</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="notCompletedListView"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
